recipes/vulkan-utility-libraries/all/conanfile.py

```diff
@@ -0,0 +1,178 @@
+            del self.options.fPIC
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def requirements(self):
+        self.requires(f"vulkan-headers/{self.version}", transitive_headers=True)
+
+    def validate(self):
+        if self.settings.compiler.get_safe("cppstd"):
+            check_min_cppstd(self, self._min_cppstd)
+
+        def loose_lt_semver(v1, v2):
+            return all(int(p1) < int(p2) for p1, p2 in zip(v1.split("."), v2.split(".")))
+
+        minimum_version = self._compilers_minimum_version.get(str(self.settings.compiler), False)
+        if minimum_version and loose_lt_semver(str(self.settings.compiler.version), minimum_version):
+            raise ConanInvalidConfiguration(
+                f"{self.ref} requires C++{self._min_cppstd}, which your compiler does not support.",
+            )
+
+    def build_requirements(self):
+        self.tool_requires("cmake/[>=3.17.2 <4]")
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], strip_root=True)
+
+    def generate(self):
+        env = VirtualBuildEnv(self)
+        env.generate()
+        tc = CMakeToolchain(self)
+        tc.variables["BUILD_TESTS"] = False
+        tc.generate()
+        deps = CMakeDeps(self)
+        deps.generate()
+
+    def _patch_sources(self):
+        replace_in_file(self, os.path.join(self.source_folder, "CMakeLists.txt"),
+                        "set(CMAKE_POSITION_INDEPENDENT_CODE ON)", "")
+
+    def build(self):
+        self._patch_sources()
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def package(self):
+        copy(self, "LICENSE.md", self.source_folder, os.path.join(self.package_folder, "licenses"))
+        cmake = CMake(self)
+        cmake.install()
+        rmdir(self, os.path.join(self.package_folder, "lib", "cmake"))
+        rm(self, "*.pdb", os.path.join(self.package_folder, "bin"))
+        fix_apple_shared_install_name(self)
+
+    @property
+    def _exported_cxxflags(self):
+        # https://github.com/KhronosGroup/Vulkan-Utility-Libraries/blob/vulkan-sdk-1.3.268.0/src/CMakeLists.txt#L9-L40
+        cxxflags = []
+        if self.settings.compiler in ["gcc", "clang", "apple-clang"]:
+            cxxflags += [
+                "-Wpedantic",
+                "-Wunreachable-code",
+                "-Wunused-function",
+                "-Wall",
+                "-Wextra",
+                "-Wpointer-arith",
+                "-Wextra-semi",
+            ]
+            if self.settings.compiler != "gcc":
+                cxxflags += [
+                    "-Wunreachable-code-return",
+                    "-Wconversion",
+                    "-Wimplicit-fallthrough",
+                    "-Wstring-conversion",
+                ]
+        elif is_msvc(self):
+            cxxflags += [
+                "/W4",
+                "/we5038",
+                "/permissive-",
+                "/MP",
```

Comment:
```suggestion
```

The Conan configuration https://docs.conan.io/2/reference/tools/build.html#conan-tools-build-build-jobs should be used instead.